core/src/main/java/org/apache/iceberg/SystemConfigs.java

```diff
@@ -72,6 +72,19 @@ private SystemConfigs() {}
           8,
           Integer::parseUnsignedInt);
 
+  /** @deprecated will be removed in 2.0.0; use name mapping instead */
+  @Deprecated
+  public static final ConfigEntry<Boolean> NETFLIX_PARQUET_ID_FALLBACK_ENABLED =
```

Comment:
Would "UNSAFE_PARQUET_ID_FALLBACK_ENABLED" be a better name (same for the property name)?

Reply:
I can add UNSAFE in there.